src/vcpkg-utils.ts

```diff
@@ -179,6 +167,9 @@ export class Utils {
   public static getAllCachedPaths(baselib: baselib.BaseLib, vcpkgRootDir: string): string[] {
     let paths = runvcpkglib.getOrdinaryCachedPaths(vcpkgRootDir);
 
+    const vcpkg_installed_path = path.resolve(vcpkgRootDir + "/../vcpkg_installed");
+    paths.push(vcpkg_installed_path);
```

Comment:
This would better be added to the `getOrdinaryCachedPaths` function in the runvcpkglib but I do not know how to edit that library and use that branch in this action for testing. This is my first time using Node other than quick tests with the REPL and my first time writing TypeScript. So maybe you could take care of changing the library and cherry pick just the first commit of this branch. In the meantime I will keep using this branch for Tenacity.